R/bcdc_search.R

```diff
@@ -236,3 +236,24 @@ as.bcdc_recordlist <- function(x) {
   class(x) <- "bcdc_recordlist"
   x
 }
+
+
+#' Provide metadata for all the resources from a single B.C. Data Catalogue record
+#'
```

Comment:
I think we need a description here as well. Something that emphasizes the rectangular structure.